debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/KafkaConnectRecordParser.java

```diff
@@ -78,13 +85,30 @@ public Record parseRecord(Object keyObj, Object valueObj) {
             Struct key = (Struct) ((SourceRecord) valueObj).key();
 
             if (value == null) {
-                // Ideally, we should have config tombstones.on.delete=false. In case that is not set correctly,
+                // Ideally, we should have config tombstones.on.delete=false. In case that is
+                // not set correctly,
                 // we will get those events where value field = null. Skipping those events.
                 LOGGER.warn("Empty value field in event. Assuming tombstone event. Skipping - {}", valueObj);
                 r.op = "unsupported";
                 return r;
+            } else if (value.schema().field("source") == null) {
+                // This is a tombstone event
+                LOGGER.warn("Empty source field in event. Assuming tombstone event. Skipping - {}", valueObj);
+                r.op = "unsupported";
+                return r;
             }
+
             Struct source = value.getStruct("source");
+
+            // Check source databse type
+            if (sourceType.equals("oracle")) {
+                // Extract transaction struct from value if it is available
+                Struct transaction = value.getStruct("transaction");
+                if (transaction != null) {
+                    r.cacheMetadata = transaction.toString();
```

Comment:
Instead of calling it as `cacheMetadata`, just make it `eventID` (of type `String`)

Reply:
Done